lib/mozart_fetcher/component.ex

```diff
@@ -67,7 +67,6 @@ defmodule MozartFetcher.Component do
   end
 
   defp metric(id, endpoint, status) when is_integer(status) do
-    ExMetrics.increment("error.component.process")
```

Comment:
this will need a PR to remove this from grafana

Reply:
This still exists for the other metric method where we metric for a reason. It's the same one, but a bit confusing it does both.

Reply:
sorry i didn't spot this metric was still being used